src/collectors/mesos/mesos.py

```diff
@@ -0,0 +1,68 @@
+# coding=utf-8
+
+"""
+
+Collects metrics from a mesos instance (master or slave).
+
+#### Example Configuration
+
+Typically metrics for the master are exposed on port 5050,
+while metrics for the slaves are exposed on port 5051.
+
+```
+    host = localhost
+    port = 5050
+```
+"""
+
+import diamond.collector
+import json
+import urllib2
+
+
+class MesosCollector(diamond.collector.Collector):
+
+    METRICS_PATH = "metrics/snapshot"
+
+    def get_default_config_help(self):
+        config_help = super(MesosCollector,
+                            self).get_default_config_help()
+        config_help.update({
+            'host': 'Hostname',
+            'port': 'Port',
+            'path': 'Metrics Path'
+        })
+        return config_help
+
+    def get_default_config(self):
+        config = super(MesosCollector, self).get_default_config()
+        config.update({
+            'host': 'localhost',
+            'port': 5050,
```

Comment:
Is there ever a case where the mesos master isn't running on 5050 and slave running on 5051? Would maybe be nice to just have a boolean to indicate if this is a master or a slave and choose the ports automatically, if we can assume that.